paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -203,3 +216,37 @@ internal fun PaymentMethodUI(
         }
     }
 }
+
+@Composable
+private fun PaymentMethodIconUi(
+    iconRes: Int,
+    iconUrl: String?,
+    imageLoader: StripeImageLoader,
+    tintOnSelected: Boolean,
+    color: Color,
+) {
+    val colorFilter = if (tintOnSelected) ColorFilter.tint(color) else null
+
+    if (iconUrl != null) {
+        StripeImage(
```

Comment:
Can we add a placeholder here, which will avoid jumpyness on slow connections? `StripeImage` has a `loadingContent` argument for that.

Reply:
Can we make the placeholder visible (e.g. a light gray box with rounded corners)?

Also, I’m noticing that the name for payment methods with remote assets isn’t fully vertically aligned with the text for payment methods with local assets, meaning that the remote assets might take up a bit more space than the local ones. Should we add a box around `StripeImage` and `Image` and enforce the size there?

Reply:
I used a blank placeholder to match what iOS is doing. If you feel strongly about changing it, I'm happy to change it and talk to wooj about changing the iOS side.

His justification for doing it this way is we still present a UI that is usable and not showing the image isn't the end of the world because we still show the payment method name